Olapic/js.class
# AMD wrapper and getter pattern removed

#### What's this PR do?

It wraps the code to use on AMD context and removes the _getter pattern_ so it won't crash with our implementation.
#### How should this be manually tested?

Load it on the browser and try the snippets on the [readme](https://github.com/Olapic/js.class/blob/master/README.md), it should work.
